spec/examples/teacher_training_api/course_single_response.json

```diff
@@ -14,6 +14,7 @@
       "accredited_body_code": "2FR",
       "age_minimum": 11,
       "age_maximum": 14,
+      "application_status": "opened",
```

Comment:
I guess the feature flag makes it awkward to also test by removing `vacancy_status` from https://github.com/DFE-Digital/apply-for-teacher-training/blob/541bd9373b2a88ee6aca8d1c3db930c664a41912/spec/examples/teacher_training_api/site_list_response_with_full_time_vacancies.json and others?

Reply:
Yes, l'll remove them :)